spiceaidocs/docs/components/data-connectors/databricks.md

```diff
@@ -34,109 +30,146 @@ datasets:
-- `databricks_azure_storage_endpoint`: Optional. The endpoint for the Azure Blob storage account.
+| Parameter Name                           | Description                                                            |
+| ---------------------------------------- | ---------------------------------------------------------------------- |
+| `databricks_azure_storage_account_name`  | The Azure Storage account name.                                        |
+| `databricks_azure_storage_account_key`   | The Azure Storage key for accessing the storage account.        |
+| `databricks_azure_storage_client_id`     | The Service Principal client ID for accessing the storage account.     |
+| `databricks_azure_storage_client_secret` | The Service Principal client secret for accessing the storage account. |
+| `databricks_azure_storage_sas_key`       | The shared access signature key for accessing the storage account.     |
+| `databricks_azure_storage_endpoint`      | Optional. The endpoint for the Azure Blob storage account.             |
+
+### Google Storage (GCS)
+
+| Parameter Name           | Description                                                  |
+| ------------------------ | ------------------------------------------------------------ |
+| `google_service_account` | Filesystem path to the Google service account JSON key file. |
+
+## Examples
+
+### Spark Connect
+
+```yaml
+  params:
+    mode: spark_connect
+    databricks_endpoint: dbc-a1b2345c-d6e7.cloud.databricks.com
+    databricks_cluster_id: 1234-567890-abcde123
+    databricks_token: ${secrets:my_token}
+```
+
+### Delta Lake (S3)
+
+```yaml
+  params:
+    mode: delta_lake
+    databricks_endpoint: dbc-a1b2345c-d6e7.cloud.databricks.com
+    databricks_token: ${secrets:my_token}
+    databricks_aws_region: us-west-2 # Optional
+    databricks_aws_access_key_id: ${secrets:aws_access_key_id}
+    databricks_aws_secret_access_key: ${secrets:aws_secret_access_key}
+    databricks_aws_endpoint: s3.us-west-2.amazonaws.com # Optional
+```
+
+### Delta Lake (Azure Blobs)
+
+```yaml
+  params:
+    mode: delta_lake
+    databricks_endpoint: dbc-a1b2345c-d6e7.cloud.databricks.com
+    databricks_token: ${secrets:my_token}
+
+    # Account Name + Key
+    databricks_azure_storage_account_name: my_account
+    databricks_azure_storage_account_key: ${secrets:my_key}
+
+    # OR Service Principal + Secret
+    databricks_azure_storage_client_id: my_client_id
+    databricks_azure_storage_client_secret: ${secrets:my_secret}
+
+    # OR SAS Key
+    databricks_azure_storage_sas_key: my_sas_key
+```
+
+### Delta Lake (GCP)
+
+```yaml
+  params:
+    mode: delta_lake
+    databricks_endpoint: dbc-a1b2345c-d6e7.cloud.databricks.com
+    databricks_token: ${secrets:my_token}
+    databricks_google_service_account_path: /path/to/service-account.json
+```
+
+## Using secrets
 
-#### Google Storage (GCS)
+There are currently three supported [secret stores](/components/secret-stores/index.md):
```

Comment:
Aren't all of the secret stores supported?

Reply:
Fixed